### PR TITLE
Adding basic menus logic and gamemanager

### PR DESCRIPTION
**Pull request by dudash**
Some really basic menus and game manager integration (currently just score tracking)

Still needs a prettier theme, still needs more options (thinking - config settings and game over).

Addresses #16 and #7

### CODE REVIEW

**Comment by rluders** [comment]
lgtm

**Review by rluders — CHANGES_REQUESTED** [review]
Could you please set the `Scenes\Main.tscn` as the initial game scene?

File: project.godot [hunk 1]
```diff
@@ -16,7 +16,7 @@ _global_script_class_icons={
 [application]
 
 config/name="PodEscape"
-run/main_scene="res://src/Levels/World.tscn"
+run/main_scene="res://src/Scenes/Main.tscn"
```
Comment (dudash) [inline project.godot:19]: @rluders 👀

**Comment by dudash** [comment]
@rluders it should be already

**Review by rluders — APPROVED** [review]

**Comment by rluders** [comment]
lgtm

File: project.godot (hunk 1 above)
Comment (rluders) [reply project.godot:19]: After I reopen the project it got it right. Crazy yah?

**Comment by RoddieKieley** [comment]
Tested locally both a regular macOS build and a HTML5 build. Works good as a regular app, both to start the game and Quit. In the html5 build if you hit Quit it doesn't appear to do anything but then when you go to Play PodEscape you simply get a blank screen. I'm guessing some resources have been released that would be required.

Not sure if the correct action in a html5 build when 'Quit' is clicked is to close the tab, which is potentially not possible, or to detect that you are in a html5 build and disable the button.

@rluders @dudash What do you think?

**Review by RoddieKieley — COMMENTED** [review]
See previous comment; also good call on the mono_crash addition to .gitignore.

**Comment by rluders** [comment]
@RoddieKieley nice catch about the quit button when the project is exported to HTML. It would be really nice if we could detect if the game was exported to HTML and remove the button in this case. But I have no idea how to do it, or even if its possible.

Maybe this could help: https://docs.godotengine.org/en/3.2/classes/class_os.html

**Comment by dudash** [comment]
Yeah, looks like we could use `OS.get_name()`. Or just pull the Quit button out completely. Is our target platform going to be HTML?

**Comment by dudash** [comment]
@RoddieKieley OK, Quit button is now hidden in HTML5 mode - that was easy. :)

**Pull request merged**